packages/core/src/node/process/process.ts

```diff
@@ -19,7 +20,7 @@ export interface IProcessExitEvent {
 
 @injectable()
 export abstract class Process implements Disposable {
-
+    abstract id: number;
     abstract readonly type: 'Raw' | 'Terminal';
     abstract pid: number;
     killed = false;
```

Comment:
`protected`?

Reply:
no I think it's useful to like print the id of a process in the logs etc... it should be safe with the readonly too